rules/core/length-zero-no-unit/README.md

```diff
@@ -0,0 +1,19 @@
+#### 📍 length-zero-no-unit
+
+Disallows units for zero lengths.
+
+##### 🧟 Example of incorrect code for this rule:
```

Comment:
please change to new file structure. see https://github.com/netsells/stylelint-config-netsells/pull/17 for an example